docs/source/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+.. _getting_started:
+
+===============
+Getting started
+===============
```

Comment:
Slight tweaks to the header formatting:

1. For consistency, using sentence casing for headings. This is common in many style guides for technical writing. I'm using a linter with some rules to help enforce [Microsoft's style guide](https://learn.microsoft.com/en-us/style-guide/welcome/) for consistency. Choosing to use Microsoft's style was somewhat arbitrary, and I don't strictly follow their rules around passive voice or using first person. I'm open to other style guides if someone has opinions.

2. Following the [recommended rules](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections) for formatting header levels.